data/tgst5.dat

```diff
@@ -11944,7 +11944,7 @@
 {"dn": 142, "lib": "", "sp": "io/vov/vitamio/utils", "bh": 391131, "btc": 285, "btn": 89}
 {"dn": 1438, "lib": "ge;badlogic;http://www.badlogicgames.com/", "sp": "com/badlogic/gdx/graphics/g3d", "bh": 609178, "btc": 271, "btn": 89, "pn": "com/badlogic"}
 {"dn": 504, "lib": "", "sp": "com/qbiki/modules/messenger", "bh": 566626, "btc": 271, "btn": 89}
-{"dn": 109, "lib": "", "sp": "org/apache/tools/zip", "bh": 163216, "btc": 263, "btn": 89}
+{"dn": 109, "lib": "ut;Apache Ant™;https://ant.apache.org/", "sp": "org/apache/tools/zip", "bh": 163216, "btc": 263, "btn": 89, "pn": "org/apache/ant"}
```

Comment:
According to the Apache Ant site (see url included), they are. Confused me at first as well, but check the ApiDocs for yourself :)